samples/snippets/src/test/java/com/example/bigquerydatatransfer/ListTransferConfigsIT.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquerydatatransfer;
+
+import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertNotNull;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.DatasetInfo;
+import com.google.cloud.bigquery.datatransfer.v1.TransferConfig;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ListTransferConfigsIT {
+
+  private BigQuery bigquery;
+  private ByteArrayOutputStream bout;
+  private String name;
+  private String displayName;
+  private String datasetName;
+  private PrintStream out;
+
+  private static final String PROJECT_ID = requireEnvVar("GOOGLE_CLOUD_PROJECT");
+
+  private static String requireEnvVar(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        "Environment variable " + varName + " is required to perform these tests.",
+        System.getenv(varName));
+    return value;
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_CLOUD_PROJECT");
+  }
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
```

Comment:
You might wish to capture the existing **Out** here

Reply:
better would be to do it in a` @Before `

Reply:
This is in `@Before`

Reply:
No before is the right place - But it should still capture Out and restore it

Reply:
We're staying consistent with other samples. e.g. https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/automl/cloud-client/src/test/java/com/example/automl/ImportDatasetTest.java

Please provide us with an example if this is not satisfactory.

Reply:
And that's the problem - other samples are doing it wrong - we just had a case where someone was doing this and then a new test was added and they no longer had access to stdout.  Restoring the old value works.

Reply:
The maven plugin running the test is providing stdout

Reply:
Done